tests/bluetooth/bsim_bt/edtt_ble_test_app/hci_test_app/prj_dut.conf

```diff
@@ -28,7 +28,4 @@ CONFIG_BT_CTLR_DATA_LENGTH_MAX=60
 
 # Enabling BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED requires BT_CTLR_ADVANCED_FEATURES
 CONFIG_BT_CTLR_ADVANCED_FEATURES=y
-CONFIG_BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED=y
-
-# LLCP Refactored controller does not support Advanced Scheduling yet
-CONFIG_BT_CTLR_SCHED_ADVANCED=n
+CONFIG_BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED=n
```

Comment:
CONFIG_BT_LL_SW_SPLIT_LLCP_CHOICE_REFACTORED no longer

Reply:
Good catch, and here I was thinking I caught them all :(